src/FormBuilder/ViewModels/FormBuilderViewModel.php

```diff
@@ -66,6 +67,7 @@ public function storageData(int $donationFormId): array
             ],
             'formFieldManagerData' => [
                 'isInstalled' => defined('GIVE_FFM_VERSION'),
+                'isLicensed' => (bool) Give_License::get_license_by_plugin_dirname('give-form-field-manager'),
```

Comment:
I think this returns data about the license.  Do we need to check if the license is valid or anything?

Reply:
Good point, yea we should. 55d9685cc6a0584c3f770be1a9dd626ef43ee257